.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: QBox Discord Server
```

```suggestion
  - name: Qbox Discord Server
```
Should have a lowercase `b` just like everywhere else.

---

qb-lapraces also has that :o

---

Absolute bozo

---

i copied the thing from the core so y'all are bozos :)

---

Incorrect, core still says `QBCore`

---

> Incorrect, core still says `QBCore`

even worse, anyway i copied the one from the core thinking you'd have updated it, turned out it didn't you kindly pointed out (and almost everytime made the changes yourself 👍🏻 ) all of them but i didn't change the other PRs i had at the same time so i guess ima update all of the repos with the right .github now :)

---

Btw when I said bozo I was referring to @ChatDisabled since he is the one that merged it 🤨

---

oh yeah i got that but i kinda took it too cauz afterall im the one writting that lol